### PR TITLE
NOISSUE: (R1) Any holder + Excerpt

### PR DESCRIPTION
**Pull request by cyraxred**

### CODE REVIEW

File: ledger-core/rms/proto_gen.go [hunk 1]
```diff
@@ -7,4 +7,4 @@ package rms
 
 // `go:generate` command below is here just for information. It should not be used by go generate.
 // Actual generation is in the Makefile.
-// go:generate protoc -I=. -I=$GOPATH/src --ins_out=./ proto_*.proto
+//go:generate protoc -I=. -I=$GOPATH/src --ins_out=./ proto_*.proto
```
Comment (sexyhamster) [inline ledger-core/rms/proto_gen.go:10]: either comment above is outdated, or this string has to have space before

Comment (cyraxred) [reply ledger-core/rms/proto_gen.go:10]: that's artifact of my local work - pls ignore ;) WIP

File: ledger-core/rms/any.go [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+	case thatValue == nil:
+		return p.value == nil
+	case p.value == nil:
+		return false
+	}
+
+	if eq, ok := thatValue.(interface{ Equal(that interface{}) bool}); ok {
+		return eq.Equal(p.value)
+	}
+	return false
```
Comment (ruz) [inline ledger-core/rms/any.go:103]: should we panic here? Uncomparable values?

File: ledger-core/rms/any_lazy.go [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+
+func (p *AnyLazy) Get() LazyValue {
+	if vv, ok := p.value.(LazyValue); ok {
+		return vv
+	}
+	return LazyValue{}
+}
+
+func (p *AnyLazy) Set(v GoGoSerializable) {
+	p.value = v
```
Comment (ruz) [inline ledger-core/rms/any_lazy.go:30]: is it impossible to check for LazyValue when we set? Set/Get sequence returns silently empty LazyValue for incorrect type. I would have placed some runtime asserts.

File: ledger-core/rms/any.go (hunk 2 above)
Comment (cyraxred) [reply ledger-core/rms/any.go:103]: it is a general notation that Equal should return false then

File: ledger-core/rms/any_lazy.go (hunk 3 above)
Comment (cyraxred) [reply ledger-core/rms/any_lazy.go:30]: well a lazy container is usually asymmetrical - set is for not-serialized value, and read for serialized / lazy one. Will correct naming to make the difference explicit

Comment (cyraxred) [reply ledger-core/rms/any_lazy.go:30]: done
https://github.com/insolar/assured-ledger/pull/386/commits/c9461a556377796c6d9ea2d51d618c6aa19302fb

**Pull request closed without merging**